src/views/bouquets/BouquetEditView.vue

```diff
@@ -1,117 +1,476 @@
+    const theme = themes.value.find((t) => t.name === selectedTheme.value)
+    return theme ? theme.subthemes : []
+  })
+
+  const themeOptions = computed(() => {
+    return themes.value.map((theme) => ({
+      text: theme.name
+    }))
+  })
+
+  const subThemeOptions = computed(() => {
+    return subThemes.value.map((subTheme) => ({
+      text: subTheme.name
+    }))
+  })
+
+  const isDatasetPropertyEmpty = computed(() => {
+    return datasetProperty.value[`${config.universe.name}:datasets_properties`].length
+  })
+
+  const description = computed(() => descriptionFromMarkdown(form))
+
+  const onThemeChanged = () => {
+    selectedSubTheme.value = null; 
+  }
+
+  const search = async (query) => {
+    if (!query) return []
+    const results = await searchAPI._search(query, config.universe.topic_id, 1,{ page_size: 10 })
+    return results.data.map(r => {
+      return { value: r.id, label: r.title, uri: r.uri }
+    }).filter(r => !datasets.value.map(d => d.dataset.id).includes(r.value))
+  }
+
+  const extrasFromDatasets = () => {
+    const extras = {}
+    for (const dataset of datasets.value) {
+      extras[`${config.universe.name}:${dataset.dataset.id}:description`] = dataset.description
+    }
+    return extras
+  }
+
+  let nextId = 0
+  const addDatasetPropertysToExtras = async () => {
+    const setUri = urlData.value ? urlData.value : null
+
+    if (libelle.value && raison.value) {
+      if (selectedDataset.value.id) {
+        const dataset = await datasetStore.load(selectedDataset.value.id)
+
+        if (dataset) {
+          datasets.value.push({ dataset, description: selectedDataset.value.description })
+
+          const getUrl = datasets.value.reduce((acc, url) => acc + url.dataset.uri, '')
+
+          datasetProperty.value[`${config.universe.name}:datasets_properties`].push({
+            libelle: libelle.value,
+            raison: raison.value,
+            uri: getUrl,
+            datagouvId: selectedDataset.value.id,
+            id: selectedDataset.value.id
+          })
+        }
+      } else {
+        datasetProperty.value[`${config.universe.name}:datasets_properties`].push({
+          libelle: libelle.value,
+          raison: raison.value,
+          uri: setUri,
+          datagouvId: null,
+          id: nextId++
+        })
+      }
+    }
+
+    libelle.value = ''
+    raison.value = ''
+    urlData.value = ''
+    datasets.value =  []
+    selectedDataset.value = {}
+    isEditDesc.value = false
+  }
+
+  const onDeleteDataset = (datasetId) => {
+    if (datasetProperty.value && datasetProperty.value[`${config.universe.name}:datasets_properties`]) {
+    datasetProperty.value[`${config.universe.name}:datasets_properties`] = datasetProperty.value[`${config.universe.name}:datasets_properties`].filter((d) => d.id !== datasetId);
+  }
+  if (loadedBouquet.value.extras) {
+    delete loadedBouquet.value.extras[`${config.universe.name}:${datasetId}:description`];
```

Comment:
Pourquoi cette propriété n'est pas dans `datasets_properties` ?

Reply:
I fixed it